addons/account/models/account_move.py

```diff
@@ -76,6 +76,8 @@ def _get_default_invoice_incoterm(self):
 
     # ==== Business fields ====
     name = fields.Char(string='Number', required=True, readonly=True, copy=False, default='/')
+    rate = fields.Float(digits=(12, 6), help='The between this currency and company currency')
```

Comment:
the rounding should be from decimal_precision engine.

Reply:
I've used the same rounding as in odoo [currency model](https://github.com/odoo/odoo/blob/250b1d3c3f976331743b10202cf30754cc396e6a/odoo/addons/base/models/res_currency.py#L238)

Reply:
There has to be a "Currency Rate" precision which is configurable. It is missing in the core and we are currently handling it with homegrown currency_rate_precision module.